app/views/layouts/application.html.erb

```diff
@@ -15,14 +15,19 @@
   <%= render partial: 'shared/assets' %>
 </head>
 
-<body class="<%= (controller.action_name == 'manage_metadata' || controller.action_name == 'manage_cmr' ? 'dashboard' : '') %>">
```

Comment:
no - we have the Manage Cmr and Manage Metadata Controllers now, with their own layouts, and the controller actions are `show`